app/models/concerns/hydra/works/collection_behavior.rb

```diff
@@ -0,0 +1,94 @@
+module Hydra::Works
+  module CollectionBehavior
```

Comment:
The code in this module depends on Hydra::PCDM::CollectionBehavior, so it should include it.  Then remove Hydra::PCDM::Collection as the parent class for Hydra::Works::Collection. It should use ActiveFedora::Base instead.